modular_citadel/code/modules/reagents/reagents/cit_reagents.dm

```diff
@@ -1,7 +1,7 @@
 //body bluids
 /datum/reagent/consumable/semen
 	name = "Semen"
-	description = "Sperm from some animal. I bet you'll drink this out of a bucket someday."
```

Comment:
oh yeah that was changed in the pr too and god this is untasteful
am i just anti fun? :+1: or :-1:

Reply:
https://github.com/Citadel-Station-13/Citadel-Station-13/pull/5828/files#diff-abb1031157e2ebd499dfd384b4d8da9eR5
Been Oldest commit shows its been their for ages

Reply:
i did not like the comment in the description here anyway

Reply:
Whatever, the current comment is too vulgarly smut, good revert.